origin-dapp/public/swAnalytics.js

```diff
@@ -0,0 +1,89 @@
+// Based off
+// https://developers.google.com/web/ilt/pwa/integrating-analytics
+
+class Analytics {
+    constructor(){
```

Comment:
@DanielVF is this file's indentation style intentional or should I update it to match the others?

Reply:
You should update it - sorry.